pom.xml

```diff
@@ -16,9 +16,9 @@
   <name>Blue Ocean Parent</name>
 
   <properties>
-    <java.level>7</java.level>
+    <java.level>8</java.level>
```

Comment:
so we're fully Java 8 now? can use all java 8 features?

Reply:
yes, once it merges java8 it is:)